cmake/iotjs.cmake

```diff
@@ -467,18 +467,18 @@ message(STATUS "EXTERNAL_LIBS            ${EXTERNAL_LIBS}")
 message(STATUS "EXTERNAL_MODULES         ${EXTERNAL_MODULES}")
 message(STATUS "IOTJS_LINKER_FLAGS       ${IOTJS_LINKER_FLAGS}")
 message(STATUS "IOTJS_PROFILE            ${IOTJS_PROFILE}")
-message(STATUS "JERRY_DEBUGGER           ${FEATURE_DEBUGGER}")
-message(STATUS "JERRY_HEAP_SIZE_KB       ${MEM_HEAP_SIZE_KB}")
-message(STATUS "JERRY_MEM_STATS          ${FEATURE_MEM_STATS}")
-message(STATUS "JERRY_PROFILE            ${FEATURE_PROFILE}")
+message(STATUS "JERRY_DEBUGGER           ${JERRY_DEBUGGER}")
+message(STATUS "JERRY_HEAP_SIZE_KB       ${JERRY_GLOBAL_HEAP_SIZE}")
```

Comment:
Is this intentionally inconsistent? JERRY_HEAP_SIZE_KB vs JERRY_GLOBAL_HEAP_SIZE

Reply:
At the moment I did not want to break the defines/options provided by IoT.js, but I think the name(s) should be normalized in a later change.

Reply:
Hm, what I mean is that the name that is printed is not the actual name of the variable that's printed. Shouldn't this be `message(STATUS "JERRY_GLOBAL_HEAP_SIZE       ${JERRY_GLOBAL_HEAP_SIZE}")`?

Reply:
Ahh sorry (was confused by the macro names), true . The original one was also misnamed, but it makes sense to use it the way you mentioned.